.github/workflows/ts-react-free2z.yml

```diff
@@ -0,0 +1,32 @@
+name: React App CI
+
+on:
+  pull_request:
+    paths:
+      - 'ts/react/free2z/**'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '20'
+
+      - name: Install dependencies
+        working-directory: ts/react/free2z
+        run: npm install
+
+      # - name: Run tests
+      #   working-directory: ts/react/free2z
+      #   run: npm test
+
+      - name: Build the project
+        working-directory: ts/react/free2z
+        # TODO: we want to set this to true soon!
```

Comment:
this is important really - we would like to fix all warnings or explicitly silence them. Perhaps we can make a bounty.